frontend/src/app/home/home.component.css

```diff
@@ -0,0 +1,6 @@
+.center-message {

```

Comment:
the design you've got here is a great start. Good job!

maybe in the future we can get into SCSS and using a design system that has white-space units baked in. I've got some patterns that I'm particularly fond of.